docs/application-connector/docs/044-remote-evironment-custom-resource.md

```diff
@@ -0,0 +1,98 @@
+---
+title: RemoteEnvironment
+type: Custom Resource
+---
+
+The `remoteenvironments.applicationconnector.kyma-project.io` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to register a Remote Environment (RE) in Kyma. The `RemoteEnvironment` Custom Resource defines the APIs that the RE offers. After creating a new Custom Resource for a given RE, the RE is mapped to appropriate ServiceClasses in the Service Catalog. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd remoteenvironments.applicationconnector.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample resource that registers the `re-prod` Remote Environment which offers one service.
```

Comment:
this sample should cover only fields that user really can use on a CR level

Reply:
I have added information about the fields that should not be edited in the "Custom resource parameters" table